src/showcase/src/showcase.ts

```diff
@@ -134,7 +134,7 @@ const authzTemplatesKnownAlt = authnTemplateAuthorize({
   i18n,
 
   knownDapp: {
-    name: "NNS Front-End Dapp",
+    name: "NNS Dapp",
```

Comment:
Should we update NNS-dapp meta and the portal as well?

Reply:
Same for the logo?

Reply:
It was updated in the portal, this is porting that change; same for the logo (see PR description)